src/models/tools/geometry/geometry-model.ts

```diff
@@ -174,7 +174,7 @@ export const PolygonModel = GeometryObjectModel
   .props({
     type: typeField("polygon"),
     points: types.array(types.string),
-    labels: types.maybe(types.array(PolygonSegmentLabelModel))
+    labels: types.array(PolygonSegmentLabelModel)
```

Comment:
Here's another example where my MST knowledge wasn't quite up to snuff. I was thinking arrays and maps are automatically `maybe`, but they aren't... they're automatically `optional`, with `[]` or `{}` being the default value, rather than `undefined` like `maybe`. Ever so slowly I'm figuring out the subtleties to MST.

In any case, my change didn't work, so I had to revert it anyway :)